src/views/v2/month.php

```diff
@@ -75,6 +75,13 @@ class="tribe-events-calendar-month"
 
 		</div>
 
+		<!-- This is the second occurrence of this, let's never show it to screen readers. -->
+		<?php
+		$this->template(
+			'components/messages', [
+			'attributes' => [ 'aria-hidden' => 'true', 'data-js-type' => 'default' ]
```

Comment:
we can probably remove `aria-hidden` here and rely on the `tribe-common-a11y-hidden` class, since we don't actually change this in the JS, it hurts us more than not.